src/ServiceControl.Management.PowerShell/Cmdlets/AuditInstances/InvokeAuditUpgrade.cs

```diff
@@ -21,6 +21,9 @@ public class InvokeServiceControlAuditInstanceUpgrade : PSCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Acknowledge mandatory requirements have been met.")]
         public string[] Acknowledgements { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Perform upgrade even if storage engine is incompatible resulting in data loss")]
+        public SwitchParameter ForceRecreateDatabase { get; set; }
```

Comment:
@DavidBoike what do you think about this name/guidance on the branch? The behavior is more like the following:
```suggestion
        [Parameter(Mandatory = false, HelpMessage = "Force upgrade to latest storage engine during upgrade. This results previous data to no longer be available. Database path will be moved as a backup. ")]
        public SwitchParameter ForceRecreateDatabase { get; set; }
```

We also already have `-Force`, we potentially could just piggy back that switch as to me Force should be a "force all".

Reply:
Totally comfortable with using the existing `-Force` rather than creating new longer param names.

Also would prefer using the actual names of the things rather than saying "latest storage engine" - whatever makes it crystal clear to the user of THIS version right now.